server/db/models/Image.js

```diff
@@ -1,12 +1,12 @@
 const conn = require('../conn');
 const { Sequelize } = conn;
 
+// used to store images for both challenges and userchallenges
+// storing images as base64 data
+
 module.exports = conn.define('image', {
   // from associations: challengeId, userchallengeId
-  type: {
-    type: Sequelize.ENUM('challenge', 'userchallenge'),
-  },
-  url: {
-    type: Sequelize.STRING,
+  data: {
```

Comment:
Have you contemplated storing the data as a binary blob? The processing time of converting base64 is more expensive than memory.

Reply:
Have never worked with them.  Will look them up.  Thanks.